libr/core/cconfig.c

```diff
@@ -3261,6 +3261,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETBPREF ("zign.autoload", "false", "Autoload all zignatures located in " R_JOIN_2_PATHS ("~", R2_HOME_ZIGNS));
 	SETPREF ("zign.diff.bthresh", "1.0", "Threshold for diffing zign bytes [0, 1] (see zc?)");
 	SETPREF ("zign.diff.gthresh", "1.0", "Threshold for diffing zign graphs [0, 1] (see zc?)");
+	SETPREF ("zign.bestthresh", "0.0", "Threshold for zb commands");
```

Comment:
I dont like the var name or the description. But it seems like an useful feature.

Proposals:

- zign.filter
- use zign.threshold. As prefix for diff.b,g,min..
- find better name instead of threshold

For the dexcription i would say zign.diff.* are also poor descriptions. Maybe better to say like “ filter fir matches having minimum threshold for bytes matching”

Reply:
I would say just `zign.threshold` with a better description than just "Threshold for zb commands".

Reply:
make clear the idfference with `	SETPREF ("zign.diff.bthresh", "1.0", "Threshold for diffing zign bytes [0, 1] (see zc?)");
`